src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +27,8 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        nextSequenceNumber++;
+        sequenceNumber = nextSequenceNumber;
```

Comment:
Semantically, the following sequence would make more sense:

```
private static int nextSequenceNumber = 1;
sequenceNumber = nextSequenceNumber;
nextSequenceNumber++;
```

By the way,
```
sequenceNumber = nextSequenceNumber;
nextSequenceNumber++;
```
can be shortened into
`sequenceNumber = nextSequenceNumber++;`

And likewise
```
nextSequenceNumber++;
sequenceNumber = nextSequenceNumber;
```
can be shortened into
`sequenceNumber = ++nextSequenceNumber;`